custom_components/tahoma/button.py

```diff
@@ -0,0 +1,73 @@
+"""Support for Overkiz number devices."""
+from homeassistant.components.button import ButtonEntity, ButtonEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ENTITY_CATEGORY_DIAGNOSTIC
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN, IGNORED_OVERKIZ_DEVICES
+from .entity import OverkizDescriptiveEntity
+
+BUTTON_DESCRIPTIONS = [
+    # My Position (cover, light)
+    ButtonEntityDescription(
+        key="my",
+        name="My Position",
+        icon="mdi:star",
+    ),
+    # Wink (cover, siren, light)
+    ButtonEntityDescription(
+        key="wink",
+        name="Wink",
+        icon="mdi:emoticon-wink",
+        entity_registry_enabled_default=False,
+    ),
```

Comment:
About this one, I got this error:

`2021-12-07 21:01:16 ERROR (MainThread) [custom_components.tahoma.executor] wink() : Invalid value for command parameter p1 : expected integer value between 1 and 254 but got no value (no type)`

Reply:
Shall I remove the wink command? Or give it a default value? I didn't test this one, I just copied it from light...

Reply:
I think we found an issue, according to the documentation:

```
{
          "commandName": "startIdentify",
          "description": "Ask the device to start winking immediately for an unlimited amount of time\n",
          "nparams": 0
        },
```

```
{
          "commandName": "stopIdentify",
          "description": "Ask the device to stop winking immediately\n",
          "nparams": 0
        },
```


```
{
          "commandName": "identify",
          "description": "Ask the device to identify itself by doing a slight movement or blink or beep\n",
          "nparams": 0
        },
```

For my cover, identify and startIdentify are reversed. I will contact Somfy Support. But I think we can keep the 3 commands. 

About wink:

```
{
          "commandName": "wink",
          "prototype": {
            "parameters": [
              {
                "optional": false,
                "sensitive": false,
                "valuePrototypes": [
                  {
                    "type": "INT",
                    "minValue": 1,
                    "maxValue": 254,
                    "description": "integer value between 1 and 254"
                  }
                ]
              }
            ]
          },
          "description": "Ask the device to identify itself by doing a slight movement or blink or beep for a given amount of seconds.\n",
          "nparams": 1
        },
```

It’s similar to identify, so IMO there is no need to keep it.

Reply:
> For my cover, identify and startIdentify are reversed. I will contact Somfy Support. But I think we can keep the 3 commands.

Cool! By the way, the `identify` command should stop automatically as well?

Reply:
That’s the point, Somfy reverse the logic. startIdentify make the cover wink a couple of time, while identify make the cover wink until you stop it. I contact their support yesterday, no answer yet.